tests/integration/components/billing/warning-message-test.js

```diff
@@ -55,7 +55,7 @@ module('Integration | Component | warning-message', function (hooks) {
       name: 'Startup',
       startingPrice: 3000,
       startingUsers: 10,
-      addon_configs: [
+      addonCsonfigs: [
```

Comment:
Looks like a typo to me. Why didn't it break anything?

Reply:
Yep, it's a typo. In this `other addons` test probably not used.

Reply:
If it's not used, would it make sense to remove it then?

Reply:
Indeed, there is a sense.